cpp/ps0000/ps0000_soin_cpp.cpp

```diff
@@ -1,8 +1,9 @@
-#include<iostream>
-#include<conio.h>
+#include <iostream> // 2/2
+#include <conio.h>
 using namespace std;
-int main()
+int main() // 1/1
 {
-	cout << "hello ,world! ";
-	_getch();
-}
\ No newline at end of file
+	cout << "Hello, World!\n"; // 3/4
+	_getch(); // 1/1
```

Comment:
- [x] `main` function returns `0` as everything is ok
⚠ I think you use VS IDE and cl.exe compiler so `return 0;` is included in `_getch()` function from the header file `conio.h`, used to fix console window so that you can see the output before typing any character from keyboard.